app/interactors/tasks/save.rb

```diff
@@ -0,0 +1,19 @@
+module Tasks
+  class Save 
+    include Interactor
+
+    delegate :task_params, :project, to: :context
+
+    def call 
+      context.project.task = task
```

Comment:
I am not sure that is ok. You get project from context and assign new task to project's task.